ee/frontend/mobile-replay/__snapshots__/transform.test.ts.snap

```diff
@@ -1689,6 +1844,13 @@ AAAAAAAAAAAAAAAAAAAAAAAAgCN/AW0xMqHnNQceAAAAAElFTkSuQmCC
             "y": 556,
           },
         },
+        {
+          "parentId": 209272202,
+          "wireframe": {
+            "id": 52129787123,
+            "type": "text",
```

Comment:
i think this is a snapshot showing this conversion not working, since it still has `wireframe` property

Reply:
ah, no, it has the transformed and untransformed data :phew: